improver/synthetic_data/set_up_test_cubes.py

```diff
@@ -420,6 +421,7 @@ def set_up_spot_variable_cube(
     realizations: Optional[Union[List[float], ndarray]] = None,
     vertical_levels: Optional[Union[List[float], ndarray]] = None,
     pressure: bool = False,
+    height: bool = True,
```

Comment:
Have we considered what happens if pressure is set to True and the height flag is left unset? It's default is True so we would have 2 True coords which will likely fail. For this to work you'd probably have to write a bit of code to handle it.

On line 370 which is the function that uses the pressure and height flags, there is a bit of code that says if pressure is set (i.e. if pressure=True) then set pressure, else use height. For this reason it might be worth just setting the default flag for height to False as it's already defaulting to height within the code.